css/src/components/gradient.scss

```diff
@@ -88,3 +88,32 @@ $border-gradient-direction: (
 	-webkit-text-fill-color: transparent;
 	line-height: 1.4; // required because of background clip, avoid "g" and "y" getting cut off
 }
+
+// Gradient card
+$gradientCardColors: (
+	'vivid': (
+		$gradient-vivid-start,
+		$gradient-vivid-end
+	)
+);
+
+// Gradient indexes used for a gradient loop
+$gradient-color-start-index: 1;
+$gradient-color-end-index: 2;
+
+@each $name, $gradientSet in $gradientCardColors {
+	$colorStart: nth($gradientSet, $gradient-color-start-index);
+	$colorEnd: nth($gradientSet, $gradient-color-end-index);
+
+	.gradient-card-#{$name} {
+		border: $border-width-md solid $border !important;
+		border-radius: $border-radius-lg !important;
+		border-color: transparent !important;
+		background: linear-gradient(var(--theme-body-background) 0 0) padding-box,
```

Comment:
can we use the scss variable here instead? I believe you'll have to use the ugly interpolation syntax, but that will give us build validation (if we remove the var or misspell it):

```
background: linear-gradient(#{$body-background } 0 0) padding-box,